ioctl/cmd/contract/contractshare.go

```diff
@@ -233,21 +234,26 @@ func share(args []string) error {
 				err := os.WriteFile(_givenPath+"/"+setPath, []byte(content), 0777)
 				if err != nil {
 					log.Println("set file failed: ", err)
+					break
 				}
 				if err := conn.WriteJSON(&response); err != nil {
 					log.Println("send set response: ", err)
 					break
 				}
-				log.Println("set: " + _givenPath + "/" + setPath)
+				log.Printf("set: %s/%s\n", _givenPath, easpcapeString(setPath))
```

Comment:
why is easpcapeString added here

Reply:
user can make many new log entries, this is unsafe.   https://github.com/iotexproject/iotex-core/security/code-scanning/10

Reply:
good. Could you reproduce this issue locally?